modules/tinymce/src/plugins/paste/main/ts/core/Utils.ts

```diff
@@ -79,7 +79,7 @@ const innerText = (html: string): string => {
     }
 
     // Walk all children
-    if (!node.shortEnded) {
+    if (!(node.name in schema.getShortEndedElements())) {
```

Comment:
```suggestion
    if (!node.name in schema.getShortEndedElements()) {
```
The parentheses aren't needed here

Reply:
<img width="516" alt="Screen Shot 2021-12-08 at 3 42 38 pm" src="https://user-images.githubusercontent.com/17724765/145154755-647890c6-da3b-4e4d-bc44-7adb1c1de3d4.png">
yes they are, otherwise it just evaluates as something like this:

```typescript
if (false in schema.getShortEndedElements()) {
```

Reply:
Right you are sorry... must have been getting confused with something else sorry.